src/MaxText/input_pipeline/_grain_data_processing.py

```diff
@@ -95,9 +95,9 @@ def pretrain_preprocessing_pipeline(dataset, config, data_columns, tokenize, gra
     dataset = dataset.map(_input_pipeline_utils.NormalizeFeatures(data_columns, tokenize))
 
   assert len(data_columns) == 1
-  rekey_dict = {"inputs": "text", "targets": "text"}
-  dataset = dataset.map(_input_pipeline_utils.Rekey(rekey_dict))
+  text_column = "text"
```

Comment:
we can have text_column = data_columns[0]? Let's move the rekey_dict definition to right before the Rekey op in line 131.

Reply:
Got it. I've updated the variable to `text_column` and moved the `rekey_dict` as requested. The changes have been pushed.